backend/database.py

```diff
@@ -141,13 +141,28 @@ class ParticipantItem(db.Model):
     )
 
 
+@dataclass
+class ParticipantAttachment(db.Model):
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    participant_id: Mapped[int] = mapped_column(ForeignKey("trip_participant.id"))
+    filename: Mapped[str] = mapped_column(nullable=True)
+    text: Mapped[str] = mapped_column()
+    created_at: Mapped[datetime] = mapped_column(default=datetime.now)
+    updated_at: Mapped[datetime] = mapped_column(
+        default=datetime.now, onupdate=datetime.now
+    )
+
+
 @dataclass
 class TripParticipant(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     user_id: Mapped[int] = mapped_column(ForeignKey("user.id"))
     user: Mapped[User] = relationship(User, backref="trip_participations")
     trip_id: Mapped[int] = mapped_column(ForeignKey("trip.id"))
     items: Mapped[list[ParticipantItem]] = relationship("ParticipantItem")
+    attachments: Mapped[list[ParticipantAttachment]] = relationship(
+        "ParticipantAttachment"
+    )
```

Comment:
Hvorfor har du valgt å legge attachments til en `TripParticipant` vs. å legge til på `Trip` direkte? Forstår at man uansett trenger en kobling til `TripParticipant` for å vite hvem som lastet opp, men tror likevel jeg ville modellert det slik at man har `TripAttachment` som er child av `Trip`. Man lager tross alt vedlegg på turen, ikke på deltakeren.

Reply:
Did i fix it now?

Reply:
Nesten, men kanskje kalle det for en `TripAttachment` ?

Reply:
Sånn?

Reply:
Good!